comicapi/comicarchive.py

```diff
@@ -50,6 +51,109 @@ class MetaDataStyle:
     COMET = 2
     name = ['ComicBookLover', 'ComicRack', 'CoMet']
 
+class SevenZipArchiver:
+
+    """7Z implementation"""
+
+    def __init__(self, path):
+        self.path = path
+
+    # @todo: Implement Comment?
+    def getArchiveComment(self):
+        return ""
+
+    def setArchiveComment(self, comment):
+        return False
+
+    def readArchiveFile(self, archive_file):
+        data = ""
+        try:
+            with py7zr.SevenZipFile(self.path, 'r') as zf:
+                data = zf.read(archive_file)[archive_file].read()
+        except py7zr.Bad7zFile as e:
+            print("bad zipfile [{0}]: {1} :: {2}".format(e, self.path,
```

Comment:
This should reference 7zip and not zipfile

Reply:
copy past error - I'll fix